app/models/creatorship.rb

```diff
@@ -41,24 +41,26 @@ def check_invalid
     end
   end
 
+  # Make sure that the user isn't banned or suspended.
+  def check_banned
+    return unless pseud&.user&.banned || pseud&.user&.suspended
+
+    errors.add(:base, ts("%{name} cannot be listed as a co-creator.",
+                         name: pseud.byline))
+    throw :abort
+  end
+
   # Make sure that if this is an invitation, we're not inviting someone who has
   # disabled invitations.
   def check_disallowed
     return if approved? || pseud.nil?
+    # Prevents this error from appearing in addition to the error for banned or suspended users. 
+    return if pseud&.user&.banned || pseud&.user&.suspended
```

Comment:
Now that you've added the `throw :abort` to `check_banned`, you should be able to delete these lines. `throw :abort` halts the execution of the callback chain, which means that it won't run any further validations (like `check_disallowed`) after `check_banned`.

Reply:
Thanks! Apologies for leaving this for so long. COVID put me down. :\ I'll get this taken care of and a new commit asap!

Reply:
Alrighty. I believe I got the change in the way you were looking for. Apologies for this PR dragging out.